testdata/fixtures/Toml/write_complex_test.toml

```diff
@@ -0,0 +1,6 @@
+a = 123
+b = "test"
+
+[c]
+a = 123
+b = "test"
```

Comment:
do we have a test for this table?

Reply:
Yes we do here, the test specifically covers the writing aspect: https://github.com/foundry-rs/foundry/blob/493c6153d0d55bb747b24f179635e7690585b176/testdata/cheats/Toml.t.sol#L259-L270